CAPABILITIES.md

```diff
@@ -0,0 +1,32 @@
+# Capabilities
+
+The Coinbase SDK has different capabilities for different wallet types and networks. This page summarizes
+those capabilities for the Ruby SDK:
+
+## Developer Wallets
+
+| Concept       | Base-Sepolia | Base-Mainnet | Ethereum-Holesky | Ethereum-Mainnet |
+| ------------- | :----------: | :----------: | :--------------: | :--------------: |
+| Addresses     |      ✅      |      ✅      |        ❌        |        ❌        |
+| Send          |      ✅      |      ✅      |        ❌        |        ❌        |
+| Trade         |      ❌      |      ✅      |        ❌        |        ❌        |
+| Faucet        |      ✅      |      ❌      |        ✅        |        ❌        |
+| Server-Signer |      ✅      |      ✅      |        ❌        |        ❌        |
+
+## End-User Wallets
+
+| Concept            | Base-Sepolia | Base-Mainnet | Ethereum-Holesky | Ethereum-Mainnet |
+| ------------------ | :----------: | :----------: | :--------------: | :--------------: |
+| External Addresses |      ❌      |      ❌      |        ✅        |        ✅        |
```

Comment:
External Address are supported on `base-sepolia` and `base-mainnet` as well for balances purposes

Reply:
Should call this "Balances" instead of Addresses in this table and above?

Reply:
Thanks for catching. Fixed.